docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -307,9 +307,16 @@ You can make further modifications to a saved filter view later using kebab menu
 
 ## Create an SLO from Metrics page
 
-:::note Coming soon
-You'll be able to create SLOs from Metrics Explorer.
-:::
+To create an SLO from the **Metrics** page:
+
+1. Go to **Manage Data** > **Metrics**.
+1. Under **Metrics Explorer**, select your desired **Metric** and **Filters**. Optionally, you can **Add Operator**.<br/><img src={useBaseUrl('img/observability/metrics-slo.png')} alt="metrics-slo.png" />
```

Comment:
In this image, "Filters" params are not fully visible. Can we use advanced mode? That will make it easier to understand.

Reply:
Does this new I've uploaded suffice?